packages/compass/package.json

```diff
@@ -205,6 +205,9 @@
           "**/node_modules/bindings/**",
           "**/file-uri-to-path/**"
         ]
+      },
+      "rebuild": {
+        "onlyModules": ["interruptor", "keytar", "kerberos"]
```

Comment:
I guess eventually we want `mongodb-client-encryption` to be a part of this list so it can be used in Compass shell, but for now this doesn't change any current behavior